packages/api/src/controllers/stream.ts

```diff
@@ -233,6 +233,28 @@ async function triggerManyIdleStreamsWebhook(ids: string[], queue: Queue) {
   );
 }
 
+export function resolvePullUrlFromExistingStreams(
+  existingStreams: DBStream[]
+): { pullUrl: string; pullRegion: string } {
+  if (existingStreams.length !== 1) {
+    return null;
+  }
+  const stream = existingStreams[0];
+  const lockLeaseTimeout = 60 * 1000; // 1 min
```

Comment:
Should we hardcode this value on the lock API as well?

Reply:
Good idea. Updated.